src/commcare_cloud/commands/terraform/entrypoint.tf.j2

```diff
@@ -108,10 +107,17 @@ module "commcarehq" {
   }
 
   account_alias = {{ (account_alias or "")|tojson }}
+}
 
-  users = [
-  {%- for user in (users if manage_users else []) %}
-    {username = {{ user.username|tojson }}},
-  {%- endfor %}
-  ]
+{%- for user in (users if manage_users else []) %}
+resource "aws_key_pair" {{ user.username|tojson }} {
+  key_name = {{ user.username|tojson }}
+  public_key = {{ user.public_key|tojson }}
+}
+
+module "iam_user__{{ user.username }}" {
```

Comment:
Why 2 underscores?

Reply:
Eh, I just figured it would be a nice way to indicate separation of the static part of the name and the dynamic part of the name.